README.md

```diff
@@ -92,14 +92,24 @@ Run the following steps on your laptop or local workstation:
         dockerflow --project=MY-PROJECT \
             --workflow-file=src/test/resources/linear-graph.yaml \
             --workspace=gs://MY-BUCKET/MY-PATH \
+	    --input BASE_DIR=gs://MY-BUCKET/MY-PATH/MY-INPUT-FILE.txt
```

Comment:
It doesn't appear that this new line `--input BASE_DIR=...` lines up with the rest of the flags.